upgrade/v2.5/README.md

```diff
@@ -75,9 +75,15 @@ Make sure you have all the `IPPools`, `BGPPeers`, `GlobalFelixConfig`, and `Glob
 
 ## 4. Upgrade calico
 
+  > Note: If you are updating a [Canal](https://github.com/projectcalico/canal/tree/master/k8s-install)
+  deployment do not use the RBAC manifest in the following step, instead use the
+  appropriate [Canal RBAC manifest](https://github.com/projectcalico/canal/blob/master/k8s-install/1.7/rbac.yaml).
+
   4.1. If you have RBAC enabled, apply the updated RBAC manifest `kubectl apply -f https://docs.projectcalico.org/v2.5/getting-started/kubernetes/installation/hosted/rbac-kdd.yaml`. (This will revoke access to TPRs from calico-node.)
   
-  4.2. Now you can upgrade the Calico version to `v2.5.0` in your Kubernetes Calico DaemonSet. (Make sure you reboot your calico-node pods one at a time if calico-node `updateStrategy` is not set to `RollingUpdate`.) 
+  4.2. Now you can install the new manifest or upgrade the calico/node and
```

Comment:
not sure I understand why we need `install the new manifest or`? This is under `4. Upgrade Calico`, installing new manifest sounds confusing. The only new thing you get with installing the new manifest is the CRDs, but they should already be created during the migration

Reply:
It avoids editing a manifest.  I thought our normal upgrade procedure is to "Install the new manifest".

Reply:
Eh, I guess not.  The Upgrade docs say to do components one at a time and don't really talk about just installing the new manifests.  I'll get rid of that.